src/rpc/blockchain.cpp

```diff
@@ -2244,7 +2280,8 @@ static RPCHelpMan scanblocks()
             scan_objects_arg_desc,
             RPCArg{"start_height", RPCArg::Type::NUM, RPCArg::Default{0}, "Height to start to scan from"},
             RPCArg{"stop_height", RPCArg::Type::NUM, RPCArg::DefaultHint{"chain tip"}, "Height to stop to scan"},
-            RPCArg{"filtertype", RPCArg::Type::STR, RPCArg::Default{BlockFilterTypeName(BlockFilterType::BASIC)}, "The type name of the filter"}
+            RPCArg{"filtertype", RPCArg::Type::STR, RPCArg::Default{BlockFilterTypeName(BlockFilterType::BASIC)}, "The type name of the filter"},
+            RPCArg{"filter_false_positives", RPCArg::Type::BOOL, RPCArg::Default{false}, "Filter false positives. Otherwise they may occur at a rate of 1/M"},
```

Comment:
Should we document that this RPC requires `blockfilterindex=1` and that it is scanning using the block filters? It doesn't seem like this is noted in the help text above. Then we can say here that block filter scanning has a rate of false positives of 1/M, and that using this option will require Bitcoin Core to read all the blocks from disk and scan them, so it will be more expensive setting this option. Also that this option can fail on a pruned node.